src/order/bounded_lattice.lean

```diff
@@ -998,6 +998,15 @@ by simp [disjoint]
 lemma disjoint.ne {a b : α} (ha : a ≠ ⊥) (hab : disjoint a b) : a ≠ b :=
 by { intro h, rw [←h, disjoint_self] at hab, exact ha hab }
 
+lemma disjoint.le_eq_bot {a b : α} (hab : disjoint a b) (h : a ≤ b) : a = ⊥ :=
+eq_bot_iff.2 (by rwa ←inf_eq_left.2 h)
+
+lemma disjoint.disjoint_inf_of_le {a b c : α} (h : disjoint (a ⊓ b) c) (hle : a ≤ c) :
```

Comment:
This lemma name implies to me that the conclusion has an inf in it

Reply:
Hmm, I'm having trouble coming up with a good name. Are there some other lemmas which specify that the arguments of some function (here `inf`) have some property?

Reply:
Maybe `disjoint.of_disjoint_inf_of_le`?

Reply:
I've gone with `disjoint.of_disjoint_inf_of_le` per @fpvandoorn's suggestion.